eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/ClusteredVariant.java

```diff
@@ -0,0 +1,180 @@
+    }
+
+    public ClusteredVariant(String assemblyAccession, int taxonomyAccession, String contig, long start,
+                            VariantType type, Boolean validated) {
+        if (Objects.isNull(assemblyAccession)) {
+            throw new IllegalArgumentException("Assembly accession is required");
+        }
+        if (Objects.isNull(contig)) {
+            throw new IllegalArgumentException("Contig is required");
+        }
+        if (Objects.isNull(type)) {
+            throw new IllegalArgumentException("Variant type is required");
+        }
+
+        this.assemblyAccession = assemblyAccession;
+        this.taxonomyAccession = taxonomyAccession;
+        this.contig = contig;
+        this.start = start;
+        this.type = type;
+        this.validated = validated;
+        this.createdDate = null;
+    }
+
+    @Override
+    public String getAssemblyAccession() {
+        return assemblyAccession;
+    }
+
+    public void setAssemblyAccession(String assemblyAccession) {
+        this.assemblyAccession = assemblyAccession;
+    }
+
+    @Override
+    public int getTaxonomyAccession() {
+        return taxonomyAccession;
+    }
+
+    public void setTaxonomyAccession(int taxonomyAccession) {
+        this.taxonomyAccession = taxonomyAccession;
+    }
+
+    @Override
+    public String getContig() {
+        return contig;
+    }
+
+    public void setContig(String contig) {
+        this.contig = contig;
+    }
+
+    @Override
+    public long getStart() {
+        return start;
+    }
+
+    public void setStart(long start) {
+        this.start = start;
+    }
+
+    @Override
+    public VariantType getType() {
+        return type;
+    }
+
+    public void setType(VariantType type) {
+        this.type = type;
+    }
+
+    @Override
+    public Boolean isValidated() {
+        return validated;
+    }
+
+    public void setValidated(Boolean validated) {
+        this.validated = validated;
+    }
+
+    @Override
+    public LocalDateTime getCreatedDate() {
+        return createdDate;
+    }
+
+    public void setCreatedDate(LocalDateTime createdDate) {
+        this.createdDate = createdDate;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof ClusteredVariant)) {
+            return false;
+        }
+
+        ClusteredVariant that = (ClusteredVariant) o;
+
+        if (taxonomyAccession != that.taxonomyAccession) {
+            return false;
+        }
+        if (start != that.start) {
+            return false;
+        }
+        if (!assemblyAccession.equals(that.assemblyAccession)) {
+            return false;
+        }
+        if (!contig.equals(that.contig)) {
+            return false;
+        }
+        if (type != that.type) {
+            return false;
+        }
+        return validated != null ? validated.equals(that.validated) : that.validated == null;
```

Comment:
I think 'validated' should not be one of the attributes identifying the variant, as is not being used in DbsnpClusteredVariantSummaryFunction. Unless we can find two different RSs in the same taxonomy, assembly, position, type... but different validation status, but then, we should think about adding it also to the summary function.